tests/sentry/grouping/ingest/test_seer.py

```diff
@@ -157,12 +170,42 @@ def test_obeys_customized_fingerprint_check(self) -> None:
             (custom_fingerprint_event, False),
             (built_in_fingerprint_event, False),
         ]:
-
             assert (
                 should_call_seer_for_grouping(event, event.get_grouping_variants())
                 is expected_result
             ), f'Case with fingerprint {event.data["fingerprint"]} failed.'
 
+    @patch("sentry.grouping.ingest.seer.metrics")
+    def test_obeys_empty_stacktrace_string(self, mock_metrics: Mock) -> None:
+        self.project.update_option("sentry:similarity_backfill_completed", int(time()))
+        new_event = Event(
+            project_id=self.project.id,
+            event_id="12312012112120120908201304152013",
+            data={
+                "title": "title",
+                "platform": "python",
+            },
```

Comment:
To actually be an example of the thing we're talking about, wouldn't we have to have a stacktrace, but one with no contributing frames or something? Wouldn't an event without a stacktrace at all already have gotten weeded out by `event_content_is_seer_eligibile`?

Reply:
yeah good point. I previously mocked the response for `event_content_is_seer_eligibile` to be true but I've changed it to an invalid stacktrace